systests/rs-sse/rs-sse-base/src/main/java/org/apache/cxf/systest/jaxrs/sse/AbstractSseBaseTest.java

```diff
@@ -58,7 +58,7 @@ protected WebClient createWebClient(final String url) {
     protected WebTarget createWebTarget(final String url) {
         return ClientBuilder
             .newClient()
-            .property("http.receive.timeout", 8000)
+            .property("http.receive.timeout", 8000000)
```

Comment:
Please revert

Reply:
Apologies - I changed this to allow debugging. Clearly a timeout of over two hours isn't advisable for automated integration tests!